test/test-render-root.js

```diff
@@ -11,21 +11,55 @@ class TestElement extends XElement {
     };
   }
 }
-customElements.define('test-element', TestElement);
+customElements.define('test-element-1', TestElement1);
+
+class TestElement2 extends XElement {
+  static get styleSheets() {
+    // TODO: Replace with direct import of css file when better-supported in
```

Comment:
You’ll have to use your imagination a bit until import attributes are better-supported. This is the pattern we’d be using though.

Reply:
We no longer need to use our imagination :^)

We might be able to update these to import the CSS in directly. Assuming all our testing clients are using Chrome, that should probably work!.

Reply:
… except for ESLint doesn’t dig this yet. So close!